taxonomies/categories.txt

```diff
@@ -71372,6 +71372,12 @@ ciqual_food_code:en:28451
 ciqual_food_name:en:Pork, loin, roasted/baked
 ciqual_food_name:fr:Porc, échine, rôtie/cuite au four
 
+<en:Pork meat
+fr:échine de porc
```

Comment:
it would be great to have an English translation. But it seems Ciqual used the same name for both 28302 and 28003.

Reply:
Yes, it seems that the English language doesn't have a distinction between Longe de porc and Echine de porc. They both fall into the "loin" category. Or they call the Echine "spare ribs", which should really be the Travers de porc.